docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience. By leveraging advanced Artificial Intelligence (AI) algorithms, this feature intelligently identifies and suppresses extraneous sounds, ensuring clear and crisp audio output.
+
+## Key Benefits
+
+- **Enhanced Audio Quality**: Eliminates unwanted noise, including background chatter, clicks, claps, barking, and other sudden audio disturbances, resulting in a more pleasant listening experience for your audience.
+
+- **Improved Clarity**: Ensures that the primary audio content remains prominent and intelligible by reducing distractions caused by ambient noise.
+
+- **Optimized Communication**: Facilitates seamless communication in conferences and live streams by minimizing disruptions caused by environmental factors, thereby enhancing the overall professionalism of the presentation.
+
+This is a guide to enabling and using the Noise cancellation on 100ms.
+
+<StepsToc
+    parentId="noise-cancellation"
+    descriptions={[
+        "General information on 100ms Noise cancellation.",
+        "Enabling the Noise cancellation for your room(s).",
+        "Exploring platforms and methods of integration.",
+        "Activating and Managing Noise cancellation."
+    ]}
+/>
+
+<StepsContainer id="noise-cancellation">
+
+### Getting Started
+100ms Noise cancellation is powered by [krisp.ai](https://krisp.ai/),ensuring clear communication by filtering out background noise.
+
+<Note type='warning'>
+**IMPORTANT** <br/>
+This is an add-on paid feature, for details check [100ms pricing page](https://www.100ms.live/pricing/) 
+</Note>
+
+### Enabling the Noise cancellation
+
+Noise cancellation can be configured from the 100ms dashboard.
+
+#### Enabling the Noise cancellation
```

Comment:
Add a note below this section that by default, noise cancellation will be enabled for everyone in the session. If they don't want that, the next section will be helpful

Reply:
got it. adding a note